### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by zebdelrosario**
@LukeElliman Please let me know your thoughts, thanks in advance

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
```
Comment (LukeElliman) [inline prac_06/car.py:10]: Fuel has a little comment about what it is, name and odometer do not.

**Comment by LukeElliman** [comment]
Overall code is good, is set out succinctly, follows conventions well from what I can tell and it is clear what the functions and methods do.

File: prac_06/car.py (hunk 1 above)
Comment (zebdelrosario) [reply prac_06/car.py:10]: Noted and amended, thank you.

**Comment by zebdelrosario** [comment]
> Overall code is good, is set out succinctly, follows conventions well from what I can tell and it is clear what the functions and methods do.

Thank you for the prompt review

**Pull request merged**